bufferobserver.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+// BufferObserver separates observer functionality out of file
```

Comment:
Ideally, comments (particularly type and function comments) should be comprised of complete sentences with periods.